_posts/2021-03-01-changelog-142.markdown

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #142"
+date: 2021-03-01 17:00:00 +0300
+---
+
+
+## New Features
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6877 -->
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6835 -->
+* Debugging is now available in [PyCharm Professional](https://www.jetbrains.com/pycharm/) and
+  [GoLand](https://www.jetbrains.com/go/) using
+  [Native Debugging Support](https://plugins.jetbrains.com/plugin/12775-native-debugging-support) plugin
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/6807 -->
+* `Change signature` (`Refactor | Change Signature` or <kbd>Ctrl+F6</kbd>) refactoring now allows you to choose a
+  default value for new parameters (by [@Kobzol])
+
+## Fixes
+
+* Improvements in [new name resolution](https://github.com/intellij-rust/intellij-rust/issues/6217) engine
+    <!-- https://github.com/intellij-rust/intellij-rust/pull/6734 -->
+    * Fix resolve macro calls inside file included by `include!` macro
```

Comment:
```suggestion
    * Fix name resolution of macro calls inside file included by `include!` macro
```

Reply:
"Resolve" is verb so let's use "name resolution" instead. Could you check all text, please?